python/ct/crypto/verify.py

```diff
@@ -347,4 +347,41 @@ def verify_sct(self, sct, chain):
 
         return self._verify(signature_input, sct.signature.signature)
 
+    def verify_embedded_scts(self, chain):
+        """Extract and verify SCTs embedded in an X.509 certificate.
+
+        Args:
+            chain: list of cert.Certificate instances.
+
+        Returns:
+            List of (SignedCertificateTimestamp, bool) pairs, one for each SCT
+                present in the certificate. The boolean is True if the
+                corresponding SCT is valid, False otherwise.
+
+        Raises:
+            ct.crypto.error.EncodingError: failed to encode signature input,
+                or decode the signature.
+        """
+
+        leaf_cert = chain[0]
```

Comment:
`IncompleteChainError` is a subclass of `VerifyError`, so it gets caught by the try-except block, and False is set inside the list.